cfme/tests/cli/test_appliance_update.py

```diff
@@ -300,6 +299,7 @@ def test_update_replicated_webui(get_replicated_appliances_with_providers, appli
 
 
 @pytest.mark.ignore_stream("upstream")
+@pytest.mark.meta(coverage=[1704835])
```

Comment:
Same as above 
@pytest.mark.meta(automates=[1704835])

Reply:
Also you can add bugzilla docblock

Reply:
thanks done